pytest_fixtures/core/sys.py

```diff
@@ -1,6 +1,7 @@
 from urllib.parse import urlparse
 
 import pytest
+from nailgun import entities
```

Comment:
Instead of importing nailgun here, you could call like `default_sat.cli.Setting().###`

Reply:
@Gauravtalreja1 close! To keep things on the api, use `default_sat.api.Setting` to reference the same class from `nailgun.entities.Setting`.